docs/sources/plugins/developing/code-styleguide.md

```diff
@@ -60,7 +60,7 @@ Minimal plugin.json:
 
 ### README.md
 
-The README.md file is rendered both on Grafana.net and in the plugins section in Grafana. The only difference from how GitHub renders markdown is that html is not allowed.
+The README.md file is rendered in the plugins section in Grafana. The only difference from how GitHub renders markdown is that html is not allowed.
```

Comment:
@DanCech, can you provide guidance here? I am pretty sure this is now the Overview tab in the Plugins page.

Reply:
It should say "rendered both in the grafana.com plugins page, and within the grafana application"  or something like that